src/ModelContextProtocol-SemanticKernel/Types/ClaudeConfig.cs

```diff
@@ -2,9 +2,9 @@
 
 namespace ModelContextProtocol.SemanticKernel.Types;
 
-public class ClaudeConfig
+public class ClaudeConfig : McpConfig<ClaudeMcpServer>
```

Comment:
I can't remember why I made this file public, it's only used internally I think. Can you change that, also for vscode?